package.json

```diff
@@ -6,17 +6,28 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "format": "prettier --check .gitignore .",
+    "format:fix": "prettier --write .gitignore ."
```

Comment:
this command prettifies only a single file ".gitignore" 
you should update `.gitignore` to `.` (`prettier --write .`) to prettify every file in the project